index.js

```diff
@@ -9875,35 +10120,6 @@ module.exports = {
             }
         },
         'p-applicant-describe-incident': {
-            l10n: {
-                vars: {
-                    lng: 'en',
-                    context: {
-                        $data:
-                            '/answers/p-applicant-who-are-you-applying-for/q-applicant-who-are-you-applying-for'
-                    },
-                    ns: 'p-applicant-describe-incident'
-                },
-                translations: [
-                    {
-                        language: 'en',
-                        namespace: 'p-applicant-describe-incident',
-                        resources: {
-                            'q-applicant-describe-incident': {
-                                error: {
-                                    required:
-                                        'Select yes if you would like to describe the crime in your own words',
-                                    'required_someone-else':
-                                        'Select yes if you would like to briefly describe the crime'
```

Comment:
have they specifically asked that this schema is reverted?

Reply:
Yeah, the question should always be asked to the applicant, so not necessary to contextualise it.